main/streams/streams.c

```diff
@@ -502,43 +502,13 @@ fprintf(stderr, "stream_free: %s:%p[%s] preserve_handle=%d release_cast=%d remov
 			/* we don't work with *stream but need its value for comparison */
 			zend_hash_apply_with_argument(&EG(persistent_list), _php_stream_free_persistent, stream);
 		}
-#if ZEND_DEBUG
-		if ((close_options & PHP_STREAM_FREE_RSRC_DTOR) && (stream->__exposed == 0) && (EG(error_reporting) & E_WARNING)) {
-			/* it leaked: Lets deliberately NOT pefree it so that the memory manager shows it
-			 * as leaked; it will log a warning, but lets help it out and display what kind
-			 * of stream it was. */
-			if (!CG(unclean_shutdown)) {
-				char *leakinfo;
-				spprintf(&leakinfo, 0, __FILE__ "(%d) : Stream of type '%s' %p (path:%s) was not closed\n", __LINE__, stream->ops->label, stream, stream->orig_path);
-
-				if (stream->orig_path) {
-					pefree(stream->orig_path, stream->is_persistent);
-					stream->orig_path = NULL;
-				}
 
-# if defined(PHP_WIN32)
-				OutputDebugString(leakinfo);
-# else
-				fprintf(stderr, "%s", leakinfo);
-# endif
-				efree(leakinfo);
-			}
-		} else {
-			if (stream->orig_path) {
-				pefree(stream->orig_path, stream->is_persistent);
-				stream->orig_path = NULL;
-			}
-
-			pefree(stream, stream->is_persistent);
-		}
-#else
```

Comment:
Why is this code dropped? Shouldn't it still work fine, as fast_shutdown is only used in non-debug?